test/e2e/framework/service/jig.go

```diff
@@ -1021,24 +1018,8 @@ func (j *TestJig) CheckServiceReachability(svc *v1.Service, pod *v1.Pod) error {
 	}
 }
 
-// CreateServicePods creates a replication controller with the label same as service. Service listens to HTTP.
+// CreateServicePods creates a replication controller with the label same as service. Service listens to TCP and UDP.
 func (j *TestJig) CreateServicePods(replica int) error {
-	config := testutils.RCConfig{
```

Comment:
I didn't see that this was used anywhere, so we can use TCP and UDP directly and simplify the polling logic using netcat only

Reply:
There are not any existing callers of `CreateServicePods()` which depend on HTTP, +1 for making the function for TCP and UDP.